api/src/test/java/org/pmiops/workbench/featuredworkspace/FeaturedWorkspaceTest.java

```diff
@@ -1,7 +1,7 @@
 package org.pmiops.workbench.featuredworkspace;
 
 import static com.google.common.truth.Truth.assertThat;
-import static com.google.common.truth.Truth8.assertThat;
```

Comment:
`Truth8` is deprecated, because its features are now part of regular `Truth` - we should remove this when we see it